pydoctor/sphinx_ext/build_apidocs.py

```diff
@@ -9,8 +9,12 @@
   - C{pydoctor_args} - an iterable with all the pydoctor command line arguments used to trigger the build.
                      - (private usage) a mapping with values as iterables of pydoctor command line arguments.
 
+  - C{pydoctor_main_branch} - The name of the main branch name. Used on RTD for latest build.
```

Comment:
```suggestion
  - C{pydoctor_main_branch} - The name of the main branch name. Used when building the C{latest} version on ReadTheDocs.
```

Reply:
This will be removed... and we will not link to main branch name... it should simplify the config.